gui/mozregui/addons_editor.py

```diff
@@ -0,0 +1,90 @@
+from PyQt4.QtCore import QAbstractListModel, QString, QModelIndex, Qt, \
```

Comment:
It seems that QString is not used - this make the travis-ci/appveyor builds unhappy, see details from https://github.com/mozilla/mozregression/pull/275 (https://travis-ci.org/mozilla/mozregression/builds/76215027#L492)